MyQueue.java

```diff
@@ -0,0 +1,53 @@
+// Time Complexity : O(N)
```

Comment:
Write time for specific functions 
Time complexity: push ( O(1)) pop ((O(n))  peek ( O(N)) empty O(1)